src/app/chatroom/chatroom.component.html

```diff
@@ -0,0 +1,90 @@
+<head>
+  <link rel="stylesheet" href="chatRoomStyleSheet.css">
```

Comment:
It is better to link .css files in the @Component annotation's metadata in the .ts file (styleUrls).